change/@fluentui-react-checkbox-2020-10-19-17-41-22-icon-rename.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
Most of these should probably be patch, not minor (though I'm not sure it matters in this case since we may not end up using all these changelogs as-is)